docs/_documentations/remotedeploy-projects-eclipse.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: docs
+title: Eclipse
+description: Creating and importing projects
+keywords: users, projects, Kubernetes, LDAP, user management, access management, login, deployment, pod, security, securing cloud connection, remote deployment of Codewind
+duration: 5 minutes
+permalink: remotedeploy-projects-eclipse
+type: document
+---
+
+# Creating and importing projects
+
+When you configure Codewind to be used remotely, you are ready to create a project that you can build and run remotely. 
+
+If you have projects that currently exist on your local computer that you would like to continue to develop remotely, follow [Import an existing local project to the cloud](##import-an-existing-local-project-to-the-cloud).
+
+## Create a new project
+
+Go to the **Codewind Explorer** view, right-click on your remote connection, and select **Create New Project**. 
+
+![Creating new remote project](./images/remotedeploy-projects-eclipse/eclipse_create-new-project.png){:width="380"}
+
+Select the template type and enter a project name. Your new project is built and after a few seconds begins running in the cloud.
+
+## Import an existing local project to the cloud
+
+Copy an existing local project, for example `myFirstNodeProject`, over to the remote Codewind deployment:
+
+1\. Right-click on your remote connection and select **Add Existing Project**:
+
+![Add existing project](./images/remotedeploy-projects-eclipse/eclipse_add-project.png){:width="366"}
+
+2\. Click **Select a project from the workspace**, and select your project: 
+
+![Add to Codewind](./images/remotedeploy-projects-eclipse/eclipse_select-project.png){:width="654"}
+
+Codewind prompts you to confirm the project type, in this case, identified as `Node.js`. Confirm the project type and click **Finish**:
+
+![Confirm Project Type](./images/remotedeploy-projects-eclipse/eclipse_confirm-project.png){:width="654"}
+
+If you add a project to the remote connection that is already deployed on the local connection, the **Project Already Deployed** dialogue appears: 
+
+![Project Already Deployed](./images/remotedeploy-projects-eclipse/eclipse_project-deployed.png){:width="524"}
+
+3\. Leave the **Remove** button selected to remove the project from the 
+local connection and click **OK**: 
+
+![Project Added](./images/remotedeploy-projects-eclipse/eclipse_build-project.png){:width="175"}
+
+4\. Codewind begins building the code and the Docker image. Moments later the project image is uploaded to Docker Hub and used by your cloud deployment to provision a new pod:
+
+![Build Success](./images/remotedeploy-projects-eclipse/eclipse_build-project-successful.png){:width="180"}
+
+`myFirstNodeProject` on `CloudName1` cluster is now running and ready.
+
+## Move a project from one connection to another 
```

Comment:
These instructions need to be called out and signposted in the introduction to this topic, in the same way as you are calling out the instructions for importing an existing local project with:

_**Note:** If you have projects that currently exist on your local computer that you would like to continue to develop remotely, follow Import an existing local project to the cloud._


The user needs to be able to read the introduction and know:
1. what they will be doing
2. where they get information for the other related actions (which, in this case, are also on the same topic - ideally these are separate topics, but we can raise a github issue for this - are you able to raise?)